python/ray/remote_function.py

```diff
@@ -202,29 +140,21 @@ def f():
         """
 
         func_cls = self
-        new_runtime_env = parse_runtime_env(runtime_env)
-
-        options = dict(
-            num_returns=num_returns,
-            num_cpus=num_cpus,
-            num_gpus=num_gpus,
-            memory=memory,
-            object_store_memory=object_store_memory,
-            accelerator_type=accelerator_type,
-            resources=resources,
-            max_retries=max_retries,
-            retry_exceptions=retry_exceptions,
-            placement_group=placement_group,
-            placement_group_bundle_index=placement_group_bundle_index,
-            placement_group_capture_child_tasks=(placement_group_capture_child_tasks),
-            runtime_env=new_runtime_env,
-            name=name,
-            scheduling_strategy=scheduling_strategy,
-        )
+
+        # override original options
+        options_from_remote_decorator = self._option_dict.copy()
```

Comment:
nit: Could we also rename `self._option_dict` to `self._options_from_remote_decorator`? It's a bit longer but I feel it's clearer?

Reply:
I decide to use `default_options` because not all of them come from `ray.remote`, especially for actors

Reply:
oh, could you give me an example?